programmers/제일-작은수-제거.js

```diff
@@ -0,0 +1,7 @@
+function solution(arr) {
+    
+    if (arr.length === 1) return [-1];
+    
+    const result = arr.filter((ele) => {return ele !== Math.min.apply(null,arr);})
```

Comment:
```suggestion
    const result = arr.filter((ele) => ele !== Math.min.apply(null,arr))
```
이렇게 간결화도 가능해요
그리고 ele 도 좋지만 el이라고 많이들 사용한답니다.
확실한 변수명이면 더더욱 좋고요